notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -18,6 +18,59 @@ https://en.wikipedia.org/wiki/List_of_science_fiction_films_of_the_2000s
 Collection: Science fiction films of the 2000s
 
 
+movie: Pitch Black (2000)
+=========================
+
+:: Title
+Pitch Black
+
+:: Description
+Pitch Black (titled The Chronicles of Riddick: Pitch Black on its DVD
+re-release) is a 2000 American science fiction action horror film directed by
+David Twohy and co-written by Twohy and brothers Ken and Jim Wheat from a
+story conceived by the latter.
+
+Synopsis: The dangerous criminal Richard B. Riddick is being transported to
+prison in a spacecraft. When the spaceship is damaged by comet debris and
+makes an emergency crash landing on an empty desert planet, Riddick escapes.
+However, when predatory alien creatures begin attacking the survivors, Riddick
+joins forces with the surviving crew and other passengers to develop a plan to
+escape the planet.
+
+:: Date
+2000-02-18
+
+:: References
+https://en.wikipedia.org/wiki/Pitch_Black_(film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+what if I were stranded on a deserted planet [The story follows the efforts of the surviving crew and passengers of a crash-landed spaceship to get off a desert planet that was inhabited by hoards of man-eating creatures.]
+desert planet [A main novelty of the film is the baked in perpetual daylight by three suns desert planet on which it is set.]
+human vs. nature [The surviving crew and passengers of the Hunter-Gratzner spaceship struggled to survive on the desert planet on which they crash-landed.]
+trust in a potential foe [Carolyn and Johns took a calculated risk by releasing the dangerous criminal Riddick and accepting him into their party. They remained suspicious and mistrustful of him throughout the course of the film, but needed his help to survive on the planet.]
+human vs. monster [Central to the story was the surviving crew and passengers of the Hunter-Gratzner spaceship trying to not get killed by the man-eating creatures that inhabited the desert planet on which they'd crash-landed.]
+monster [The desert planet was inhabited by grotesque, man-eating creatures.]
+human character stereotype [The hero of the story Riddick was a stereotypical hard-boiled, lone wolf, in-it-for-only-himself Hollywood antihero figure.]
```

Comment:
Not keen on using this when someone "feels stereotypical" but we can't put the finger on it enough to invent a theme name. My suggestion is either take it out or invent a theme-worthy name and put it as a template parameter (or write a theme def).

Reply:
Okay, let's just delete it.